pkg/cmd/server/bootstrappolicy/infra_sa_policy.go

```diff
@@ -116,7 +116,7 @@ func init() {
 				// BuildControllerFactory.buildDeleteLW
 				{
 					Verbs:     sets.NewString("get", "list", "create", "delete"),
-					Resources: sets.NewString("pods"),
```

Comment:
I'd think that if you can create a pod by default we'll want to grant you create on this virtual resource, no? And just take away the permission for online?